scripts/test/common_test.py

```diff
@@ -455,18 +456,20 @@ def test_invalid_keyset_kid_missing(self):
 
 class ValidateEntriesTestCase(unittest.TestCase):
 
+    maxDiff = None
+
     def test_validate_entries1(self):
         entries = [
-            IssuerEntry('SHC Example Issuer 1', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 2', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 3', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 4', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 5', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 6', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 7', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 8', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 9', 'https://spec.smarthealth.cards/examples/issuer'),
-            IssuerEntry('SHC Example Issuer 10', 'https://spec.smarthealth.cards/examples/issuer')
+            IssuerEntry('SHC Example Issuer 1', 'https://spec.smarthealth.cards/examples/issuer', 'https://smarthealth.cards/', None),
+            IssuerEntry('SHC Example Issuer 2', 'https://spec.smarthealth.cards/examples/issuer', 'https://spec.smarthealth.cards/', None),
+            IssuerEntry('SHC Example Issuer 3', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 4', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 5', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 6', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 7', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 8', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 9', 'https://spec.smarthealth.cards/examples/issuer', None, None),
+            IssuerEntry('SHC Example Issuer 10', 'https://spec.smarthealth.cards/examples/issuer', None, None)
         ]
 
         actual = validate_entries(entries)
```

Comment:
I don't fully understand why this validates: all `iss` are the same, would this not trigger the duplicate error?

Reply:
Agree that this is a little confusing. This is testing the `validate_entries` function, which is a wrapper that concurrently performs validation on each item in the list. It currently returns `List[ValidationResult]` (one for each input entry) and is **not** checking for duplicates (this happens before we call `validate_entries` in `validate_entries.py`). I didn't want to change the interface / semantics of this function, so I added the duplicate entries check before.

Reply:
Ok, no problem, it's just tests. Maybe add that comment?